spec/lcm/integration/support/configuration_helper.rb

```diff
@@ -0,0 +1,98 @@
+require 'fileutils'
+
+class ConfigurationHelper
+  class << self
+    CACHE_DIR = 'spec/cache/dev_projects/'
+
+    def suffix
```

Comment:
Similar code found in shared_contexts_for_lcm_specs.rb:7, configuration_helper.rb:7, lcm_helper.rb:16 (mass = 90)